src/libs/actions/connections/SageIntacct.ts

```diff
@@ -14,4 +31,172 @@ function connectToSageIntacct(policyID: string, credentials: SageIntacctCredenti
     API.write(WRITE_COMMANDS.CONNECT_POLICY_TO_SAGE_INTACCT, parameters, {});
 }
 
-export default connectToSageIntacct;
+function prepareOnyxData(policyID: string, settingName: keyof Connections['intacct']['config']['export'], settingValue: string | null) {
```

Comment:
```suggestion
function prepareOnyxDataForExportUpdate(policyID: string, settingName: keyof SageIntacctExportConfig, settingValue: string | null) {
```
I also add prepareOnyxData... on import page PR and all these functions will have conflicting names. Let's make sure they are named precisely.